### PR TITLE
idk what to call this? i guess just a helpful PR lol

### PR DESCRIPTION
**Pull request by tozzleboy**

### CODE REVIEW

File: pages/workspace/[id]/sessions/schedules.tsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ const Home: pageWithLayout<{
 		<button onClick={() => router.push(`/workspace/${router.query.id}/sessions/new`)} className="cardBtn"><p className="font-bold text-2xl leading-5 mt-1"> New session type <br /><span className="text-gray-400 font-normal text-base "> Create a new session type   </span></p> </button>
 		<p className="text-3xl font-medium mt-5">Schedules</p>
 		<div className=" pt-5 flex flex-col lg:flex-row gap-x-3 gap-y-2">
-			{!!activeSessions.length && <div className="flex flex-col w-full">
+			{!!activeSessions.length && <div className="flex flex-col w-full gap-y-3">
```
Comment (marinofranz) [inline pages/workspace/[id]/sessions/schedules.tsx:163]: Change gap margin to `2` instead of `3`.

**Review by marinofranz — CHANGES_REQUESTED** [review]
That's all that looks off to me, fix that, and I'll review again.

File: pages/workspace/[id]/sessions/schedules.tsx (hunk 1 above)
Comment (tozzleboy) [reply pages/workspace/[id]/sessions/schedules.tsx:163]: I took the gap margin from the actual sessions page.

**Review by marinofranz — APPROVED** [review]
LGTM 👍

**Pull request merged**